index.js

```diff
@@ -12,9 +15,26 @@ const sequelizeErrorParser = require('./app/services/sequelizeErrorParser');
 
 const port = process.env.PORT;
 const app = express();
+const storage = multer.diskStorage({
+  destination: function (req, file, cb) {
+    const destFolder = req.path.split('/')[2];
```

Comment:
The path you use to upload a file can change and this code won't work anymore.
Here I would only save the file in a temporary folder, `tmp`. Then, in the service responsible for the company I would call a file service that will move the file in the correct folder.